assessments/CLI_TX_KEA/earthmover.yml

```diff
@@ -0,0 +1,871 @@
+      'has_cp_col': False
+    },
+    'txkea_language_composite': {
+      'colname': 'txkea_language_composite',
+      'descriptor': 'Language Composite',
+      'subject_assess_type':'composite',
+      'subject_assessments': [
+        {'subject_assess_ref_1':[{'subject_asses_ref_colname':'txkea_vocabulary_1', 'subject_asses_ref_descriptor':'Vocabulary Wave 1'}]},
+        {'subject_assess_ref_2':[{'subject_asses_ref_colname':'txkea_listening_comp_1', 'subject_asses_ref_descriptor':'Listening Comprehension 1'}]}
+      ],
+      'subject_assess_lang': 'English',
+      'has_cp_col': False
+    },
+    'txkea_letter_names': {
+      'colname': 'txkea_letter_names',
+      'descriptor': 'Letter Names',
+      'subject_assess_type':'subject',
+      'waves':[
+        {'wave_1':[{'colname':'txkea_letter_names_txkea_letter_names_1', 'descriptor':'Letter Names Wave 1', 'has_cp_col': True}]},
+        {'wave_2':[{'colname':'txkea_letter_names_txkea_letter_names_2', 'descriptor':'Letter Names Wave 2', 'has_cp_col': True}]},
+        {'wave_3':[{'colname':'txkea_letter_names_txkea_letter_names_3', 'descriptor':'Letter Names Wave 3', 'has_cp_col': True}]}
+      ],
+      'subject_assess_lang': 'English',
+      'has_cp_col': True
+    },
+    'txkea_spelling': {
+      'colname': 'txkea_spelling',
+      'descriptor': 'Spelling',
+      'subject_assess_type':'subject',
+      'waves':[
+        {'wave_1':[{'colname':'txkea_spelling_txkea_spelling_1', 'descriptor':'Spelling Wave 1', 'has_cp_col': True}]},
+        {'wave_2':[{'colname':'txkea_spelling_txkea_spelling_2', 'descriptor':'Spelling Wave 2', 'has_cp_col': True}]},
+        {'wave_3':[{'colname':'txkea_spelling_txkea_spelling_3', 'descriptor':'Spelling Wave 3', 'has_cp_col': True}]}
+      ],
+      'subject_assess_lang': 'English',
+      'has_cp_col': True
+    },
+    'txkea_listening_comp': { 
+      'colname': 'txkea_listening_comp',
+      'descriptor': 'Listening Comp',
+      'subject_assess_type':'subject',
+      'waves':[
+        {'wave_1':[{'colname':'txkea_listening_comp_txkea_listening_comp_1', 'descriptor':'Listening Comprehension 1', 'has_cp_col': True}]},
+        {'wave_2':[
+          {'colname':'txkea_listening_comp_txkea_listening_comp_2_receptive', 'descriptor':'Listening Comp 2 Receptive', 'module': 'Receptive', 'has_cp_col': False}, 
+          {'colname':'txkea_listening_comp_txkea_listening_comp_2_expressive', 'descriptor':'Listening Comp 2 Expressive', 'module': 'Expressive', 'has_cp_col': False}
+        ]},
+        {'wave_3':[
+          {'colname':'txkea_listening_comp_txkea_listening_comprehension_3_receptive', 'descriptor':'Listening Comprehension 3 Receptive', 'module': 'Receptive', 'has_cp_col': True},
+          {'colname':'txkea_listening_comp_txkea_listening_comprehension_3_expressive', 'descriptor':'Listening Comprehension 3 Expressive - Dropped', 'module': 'Expressive', 'has_cp_col': False}
+        ]}
+      ],
```

Comment:
For context CLI Circle TX-KEA's assessment periods occur at BOY, MOY, and EOY. For each of these intervals, there is a subject column ( subject_1 - representing wave 1, subject_2 - representing wave 2...) and within each wave, there can be a set of objective assessments used to evaluate a student..

^ This structure dictated the framework of this dictionary.